### PR TITLE
Hpcc 15549 update esdl tool usage

### PR DESCRIPTION
**Pull request by rpastrana**

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review. Also comment on how useful the new messages are.

File: tools/esdlcmd/esdlcmd_core.cpp [hunk 1]
```diff
@@ -276,9 +276,9 @@ class Esdl2XSDCmd : public EsdlHelperConvertCmd
     {
         puts("Usage:");
         puts("esdl xsd sourcePath serviceName [options]\n" );
-        puts("\nsourcePath must be absolute path to the ESDL Definition file containing the" );
-        puts("EsdlService definition for the service you want to work with.\n" );
-        puts("serviceName EsdlService definition for the service you want to work with.\n" );
+        puts("\nsourcePath - Absolute path to the EXSDL Definition file (XML) containing the" );
+        puts("               EsdlService definition for the service you want to work with.\n" );
+        puts("serviceName  - EsdlService definition for the service you want to work with.\n" );
```
Comment (wangkx) [inline tools/esdlcmd/esdlcmd_core.cpp]: I think the serviceName should be 'ESDL service name defined in EXSDL Definition file'.

File: tools/esdlcmd/esdlcmd_core.cpp [hunk 2]
```diff
@@ -276,9 +276,9 @@ class Esdl2XSDCmd : public EsdlHelperConvertCmd
     {
         puts("Usage:");
         puts("esdl xsd sourcePath serviceName [options]\n" );
-        puts("\nsourcePath must be absolute path to the ESDL Definition file containing the" );
-        puts("EsdlService definition for the service you want to work with.\n" );
-        puts("serviceName EsdlService definition for the service you want to work with.\n" );
+        puts("\nsourcePath - Absolute path to the EXSDL Definition file (XML) containing the" );
```
Comment (wangkx) [inline tools/esdlcmd/esdlcmd_core.cpp]: In my opinion, the (XML) should be (.ecm). The .ecm file is an XML file. But, the (XML) makes me confused with a .xml file.

**Comment by richardkchapman** [comment]
@rpastrana Needs rebasing for a clean merge

File: tools/esdlcmd/esdlcmd_core.cpp (hunk 2 above)
Comment (rpastrana) [inline tools/esdlcmd/esdlcmd_core.cpp]: ECM is not XML, and certainly not the EXSDL required by this feature.
In 6.2.0 we'll add a feature to create the EXSDL if an ECM definition is provided.

File: tools/esdlcmd/esdlcmd_core.cpp (hunk 1 above)
Comment (rpastrana) [inline tools/esdlcmd/esdlcmd_core.cpp]: I think the intent here was to use the actual tag name (legacy) but yes the message could be more concise.

File: tools/esdlcmd/esdlcmd_core.cpp (hunk 2 above)
Comment (wangkx) [inline tools/esdlcmd/esdlcmd_core.cpp]: Are you saying that the command should be esdl xsd filename.xml serviceName, not esdl xsd filename.ecm serviceName? I tried: esdl xsd filename.ecm serviceName and worked.

Comment (rpastrana) [inline tools/esdlcmd/esdlcmd_core.cpp]: did you have filename.xml in that folder? This is the exact issue you originally reported, the XML is required, not the ECM.

**Comment by rpastrana** [comment]
@wangkx updated messages based on code review. Please review.

**Comment by wangkx** [comment]
@rpastrana looks good now.

**Comment by rpastrana** [comment]
@richardkchapman re-based, reviewed and ready to go.

**Pull request merged**